### PR TITLE
Fixes #32451 - adds convert2rhel job

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/views/foreman_ansible/job_templates/convert_to_rhel.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        dest: /usr/share/convert2rhel/subscription-manager/katello-ca-consumer-latest.noarch.rpm
+    - name: Start convert2rhel
+      command: convert2rhel -y --activationkey "<%= activation_key %>" --org "<%= @host.organization.label %>" > /root/convert2rhel.log
+<%- if input('Restart') == "yes" -%>
+    - name: Reboot the machine
+      reboot:
+        reboot_timeout: 1800
+<%- end -%>
+    - name: Update system facts
+      command: subscription-manager facts --update
```
Comment (ezr-ondrej) [inline app/views/foreman_ansible/job_templates/convert_to_rhel.erb:45]: shouldn't this go before the reboot? :thinking:

**Comment by xprazak2** [comment]
It worked pretty well, facts were uploaded after the reboot so ack from me.

File: app/views/foreman_ansible/job_templates/convert_to_rhel.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+model: JobTemplate
+job_category: Convert 2 RHEL
+provider_type: Ansible
+kind: job_template
+%>
+<%- activation_key = input('Activation Key').blank? ? "rhel#{@host.os.major}" : input('Activation Key') -%>
```
Comment (xprazak2) [inline app/views/foreman_ansible/job_templates/convert_to_rhel.erb:25]: I am wondering if this line is needed. Activation key is required, so it should never be blank.

File: app/views/foreman_ansible/job_templates/convert_to_rhel.erb (hunk 1 above)
Comment (ares) [reply app/views/foreman_ansible/job_templates/convert_to_rhel.erb:45]: no, we need the new facts after the machine is converted to RHEL

File: app/views/foreman_ansible/job_templates/convert_to_rhel.erb (hunk 2 above)
Comment (ares) [reply app/views/foreman_ansible/job_templates/convert_to_rhel.erb:25]: ah good point, I'll drop this and repush

**Comment by ares** [comment]
updated

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**